services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -145,160 +154,238 @@ public void setUp() {
+        individualReqProcessingDelay.setName(MCOL_INDV_REQ_DELAY);
+        when(this.mockCacheable.getValue(IGlobalParameter.class, MCOL_INDV_REQ_DELAY)).thenReturn(
                 individualReqProcessingDelay);
 
-        individualRequest.setRequestStatus("Forwarded");
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
+        individualRequest.setRequestStatus(FORWARDED);
 
         final IGlobalParameter connectionTimeOutParam = new GlobalParameter();
-        connectionTimeOutParam.setName("TARGET_APP_TIMEOUT");
+        connectionTimeOutParam.setName(TARGET_APP_TIMEOUT);
         connectionTimeOutParam.setValue("1000");
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_TIMEOUT")).andReturn(
+        when(this.mockCacheable.getValue(IGlobalParameter.class, TARGET_APP_TIMEOUT)).thenReturn(
                 connectionTimeOutParam);
 
         final IGlobalParameter receiveTimeOutParam = new GlobalParameter();
-        receiveTimeOutParam.setName("TARGET_APP_RESP_TIMEOUT");
-        receiveTimeOutParam.setValue("12000");
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT")).andReturn(
+        receiveTimeOutParam.setName(TARGET_APP_RESP_TIMEOUT);
+        receiveTimeOutParam.setValue(TWELVE_THOUSAND);
+        when(this.mockCacheable.getValue(IGlobalParameter.class, TARGET_APP_RESP_TIMEOUT)).thenReturn(
                 receiveTimeOutParam);
 
         this.mockConsumerGateway.individualRequest(individualRequest, 1000, 12000);
-        EasyMock.expectLastCall().andAnswer(new IAnswer<Object>() {
-            @Override
-            public Object answer() throws Throwable {
-                ((IndividualRequest) EasyMock.getCurrentArguments()[0])
-                        .setRequestStatus(IIndividualRequest.IndividualRequestStatus.ACCEPTED.getStatus());
-                // required to be null for a void method
-                return null;
-            }
-        });
-
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
+        doAnswer(invocation -> {
+            IndividualRequest argument = invocation.getArgument(0);
+            argument.setRequestStatus(IIndividualRequest.IndividualRequestStatus.ACCEPTED.getStatus());
+            return null;
+        }).when(mockConsumerGateway).individualRequest(individualRequest, 1000, 12000);
 
         final IBulkSubmission bulkSubmission = individualRequest.getBulkSubmission();
 
-        EasyMock.expect(this.mockIndividualRequestDao.queryAsCount(EasyMock.same(IndividualRequest.class), EasyMock.isA(
-            Supplier.class))).andReturn(0L);
+        when(this.mockIndividualRequestDao.queryAsCount(eq(IndividualRequest.class), isA(
+            Supplier.class))).thenReturn(0L);
 
         mockIndividualRequestDao.persist(bulkSubmission);
```

Comment:
I think this needs to be wrapped in a doAnswer() (see above for an example).  At the moment it's actually calling persist() rather than setting it up as expected behaviour.

Reply:
removed.